hydra_pywr/runner.py

```diff
@@ -180,6 +181,14 @@ def __init__(self, *args, domain="water", **kwargs):
 
         self.limit_nodes_recording = False
 
+        tmpdir = tempfile.gettempdir()
+        self.results_location = os.getenv("PYWR_RESULTS_LOCATION", tmpdir)
+        self.bucket_name = os.getenv("PYWR_RESULTS_S3_BUCKET", 'pywr-results')
+        hashkey = hashlib.sha256().hexdigest()
```

Comment:
```suggestion
        hashkey = hashlib.sha256(randbytes(56)).hexdigest()
```

`randbytes` is imported above but not used, so have added here. (56 because this gives 448 bits of randomness plus 64 bits padding is still a single block.)